cas-server-webapp/src/main/webapp/WEB-INF/webflow/login/login-webflow.xml

```diff
@@ -97,8 +97,17 @@
         <transition on="success" to="sendTicketGrantingTicket" />
         <transition on="authenticationFailure" to="handleAuthenticationFailure" />
         <transition on="error" to="viewLoginForm" />
+        <transition on="institution" to="institutionLogin" />
```

Comment:
Add `OpenScienceFrameworkLoginHandler` with action `beforeLogin` which transits to `osfLogin` or `institutionLogin`. In the future, there will be another `osfCampaignLogin` to handle campaigns except institution in general.